packages/bierzo-wallet/src/routes/payment/components/index.tsx

```diff
@@ -52,40 +19,27 @@ const Layout = ({
   onTokenSelectionChanged,
   selectedChainCodec,
 }: Props): JSX.Element => {
-  const classes = useStyles();
-
   const { form, handleSubmit, invalid, pristine, submitting } = useForm({
     onSubmit,
   });
 
   return (
     <Form onSubmit={handleSubmit}>
-      <Block
-        marginTop={4}
-        width="100%"
-        height="auto"
-        minHeight="100vh"
-        display="grid"
-        alignContent="center"
-        justifyContent="center"
-        className={classes.payment}
-      >
-        <Block width="100%" className={classes.currencyToSend}>
-          <CurrencyToSend form={form} onTokenSelectionChanged={onTokenSelectionChanged} />
-        </Block>
-        <Block width="100%" className={classes.receiverAddress}>
-          <ReceiverAddress form={form} selectedChainCodec={selectedChainCodec} />
-        </Block>
-        <Block width="100%" className={classes.textNote}>
-          <TextNote form={form} />
-        </Block>
-        <Block width="75%" className={classes.continue}>
-          <Button fullWidth type="submit" disabled={invalid || pristine || submitting}>
-            Continue
-          </Button>
-          <Back fullWidth disabled={submitting} onClick={onCancelPayment}>
-            Cancel
-          </Back>
+      <Block marginTop={5} display="flex" justifyContent="center">
+        <Block maxWidth="450px" display="flex" flexDirection="column" alignItems="center">
+          <Block minWidth="450px" bgcolor="white" padding={5}>
```

Comment:
Interesting those two css properties almost is in conflict of each other . I am talking about `maxWidth` and `minWidth`. Are you sure that they are required?